src/transformer/descriptor/indexedAccess/indexedAccess.ts

```diff
@@ -44,7 +45,8 @@ export function GetIndexedAccessTypeDescriptor(
       break;
     default:
       TransformerLogger().typeNotSupported(
-        'IndexedAccess of ' + ts.SyntaxKind[node.indexType.kind]
+        'IndexedAccess of ' + ts.SyntaxKind[node.indexType.kind],
```

Comment:
Could use template literal instead of concatenation

Reply:
Ill do that :) thanks!